phylanx/util/detail/add_simd.hpp

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2018 Shahrzad Shirzad
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef PHYLANX_UTIL_DETAIL_BLAZE_SIMD_ADD_MAY_16_2018_0500PM
+#define PHYLANX_UTIL_DETAIL_BLAZE_SIMD_ADD_MAY_16_2018_0500PM
+//
+
```

Comment:
Do you need to include some Blaze header here (for instance for `blaze::HasSIMDAdd`)?